openwisp_radius/tests/test_admin.py

```diff
@@ -660,6 +660,14 @@ def test_radiusbatch_org_user(self):
         self.assertEqual(OrganizationUser.objects.all().count(), 3)
         for u in OrganizationUser.objects.all():
             self.assertEqual(u.organization, RadiusBatch.objects.first().organization)
+        # Test CSV URL in change form
+        rad_batch = RadiusBatch.objects.first()
+        response = self.client.get(
+            reverse(f'admin:{self.app_label}_radiusbatch_change', args=[rad_batch.id])
```

Comment:
I am not sure if it will test what we want it to test, because we are using reverse, it will always resolve some path, if the path is expected or not is not tested.
In other words, I don't think this test will fail without the patch either.

About testing it, I think if we want to test it, we need to resolve the path and compare it with `CSV_URL_PATH`.

I'll check it out and try to do this approach in a couple of hours! :smile:

Reply:
The goal of this test is to ensure that the URL used in admin UI is correct, i.e. it does not return HTTP 404.
In the current master, it is not possible to download the uploaded CSV file from the admin.

Reply:
Oh, if you've checked it works, then we are good, I just assumed based on my thought that we need to compare it with `CSV_URL_PATH`! 

Thanks! :smile: